src/node.rs

```diff
@@ -153,6 +184,9 @@ impl Sprawl {
         Ok(self.ids_to_nodes[&prev_id])
     }
 
+    /// Removes the "n-th" child from the parent `node`
```

Comment:
Since we don't have an actual variable `n` here, maybe this would be more clear?
```suggestion
    /// Removes the child at the given `index` from the parent `node`
```